crates/string-offsets/Cargo.toml

```diff
@@ -8,6 +8,13 @@ repository = "https://github.com/github/rust-gems"
 license = "MIT"
 keywords = ["unicode", "positions", "utf16", "characters", "lines"]
 categories = ["algorithms", "data-structures", "text-processing", "development-tools::ffi"]
+exclude = ["/js"]
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+wasm-bindgen = "0.2"
```

Comment:
Question: do we want to have this as an unconditional dependency?

I spot checked a few other crates, and it seems more common to put it behind a feature flag or make it [conditional on a build target](https://github.com/seanmonstar/reqwest/blob/f63c6315911154de8b96b79012ce8e266e47dba6/Cargo.toml#L191-L196). However, doing it like this would make the inline attributes very anoying to write (they'd all have to be wrapped in conditionals as well), and a separate `wasm` module with the exported API that is conditionally included seems cleaner.

Reply:
Ok good idea. I moved this to use a feature so that wasm-bindgen is now an optional dependency 

I'd love to split the was stuff out to its own module but wasn't sure how to do this without duplicating a whole bunch of code